### PR TITLE
docs: ASP.NET Core WebAPI example(s)

### LINKED ISSUE #220: Docs: ASP.NET Core WebAPI example(s)
### What were you searching in the docs?

Example of how to make use of PowerTools within a C# project created using the serverless.AspNetCoreWebAPI template.

In this template there is no handler method as everything is done through a Lambda Entry Point which is a sub-class of Amazon.Lambda.AspNetCoreServer.APIGatewayProxyFunction.

### Is this related to an existing documentation section?

_No response_

### How can we improve?

```markdown
Provide more examples, or templates that make use of PowerTools.
```

### Got a suggestion in mind?

_No response_

### Acknowledgment

- [X] I understand the final update might be different from my proposed suggestion, or refused.

### PR DESCRIPTION
**Pull request by swimming-potato**
Issue number: [#220](https://github.com/awslabs/aws-lambda-powertools-dotnet/issues/220)

### Changes

Introduce an example for ASP.Net WebAPI deployed on AWS Lambda.

### User experience

This is a change in documentation. A section `ServerlessAPi` has been added to the examples. The section demonstrates to AWS Lambda Powertools users: how to leverage logging, metrics and tracing provided by AWS Lambda Powertools in the WebAPI hosted by Kestrel. It contains the code and README.md file to walkthrough the changes. 

## Checklist

Please leave checklist items unchecked if they do not apply to your change.

* [x] [Meets tenets criteria](https://awslabs.github.io/aws-lambda-powertools-dotnet/tenets)
* [ ] I have performed a self-review of this change
* [x] Changes have been tested
* [x] Changes are documented
* [x] PR title follows [conventional commit semantics](https://github.com/awslabs/aws-lambda-powertools-dotnet/blob/develop/.github/semantic.yml)

<details>
<summary>Is this a breaking change?</summary>
No
**RFC issue number**:

Checklist:

* [ ] Migration process documented
* [ ] Implement warnings (if it can live side by side)

</details>

## Acknowledgment

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Disclaimer**: We value your time and bandwidth. As such, any pull requests created on non-triaged issues might not be successful.

### CODE REVIEW

**Review by hjgraca — APPROVED** [review]

**Comment by amirkaws** [comment]
Unit test is breaking because the FunctionVersion is null


to solve that we can set the 


Amazon Copyright comments are missing from files

**Comment by swimming-potato** [comment]
HI @amirkaws I did address your comment:) please check the PR.

**Comment by amirkaws** [comment]
Hi @swimming-potato, thank you a lot for that. I think we are still missing copyright comments, we should add Amazon copyright comment on top of each .cs file. 




The other thing I believe we should add is a solution file in the example root folder to include both projects juts to be consistent with the. other examples.  here is an example for logging  https://github.com/aws-powertools/powertools-lambda-dotnet/blob/develop/examples/Logging/PowertoolsLoggingExample.sln

File: examples/ServerlessApi/ServerlessApi.sln
Comment (hjgraca) [inline examples/ServerlessApi/ServerlessApi.sln:1]: Please rename the solution to PowertoolsServerlessApiExample

**Review by hjgraca — CHANGES_REQUESTED** [review]
Looks good. Small solution name change

**Review by hjgraca — APPROVED** [review]

**Review by amirkaws — APPROVED** [review]
LGTM

**Comment by hjgraca** [comment]
Thank you for a great work @swimming-potato !

**Pull request merged**